master/src/main/scala/org/apache/celeborn/service/deploy/master/quota/QuotaManager.scala

```diff
@@ -14,35 +14,27 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-package org.apache.celeborn.common.quota
-
-import org.junit.Assert.assertEquals
+package org.apache.celeborn.service.deploy.master.quota
 
 import org.apache.celeborn.common.CelebornConf
 import org.apache.celeborn.common.identity.UserIdentifier
-import org.apache.celeborn.common.util.Utils
-
-class DefaultQuotaManagerSuite extends BaseQuotaManagerSuite {
-
-  override def beforeAll(): Unit = {
-    val conf = new CelebornConf()
-    conf.set(
-      CelebornConf.QUOTA_CONFIGURATION_PATH.key,
-      getTestResourceFile("test-quota.yaml").getPath)
-    quotaManager = QuotaManager.instantiate(conf)
-  }
-
-  test("initialize QuotaManager") {
-    assert(quotaManager.isInstanceOf[DefaultQuotaManager])
-  }
+import org.apache.celeborn.common.internal.Logging
+import org.apache.celeborn.common.quota.Quota
+import org.apache.celeborn.server.common.service.config.ConfigService
 
-  test("test celeborn quota conf") {
-    assertEquals(
-      quotaManager.getQuota(UserIdentifier("AAA", "Tom")),
-      Quota(Utils.byteStringAsBytes("100m"), 200, -1, -1))
-    assertEquals(
-      quotaManager.getQuota(UserIdentifier("BBB", "Jerry")),
-      Quota(-1, -1, Utils.byteStringAsBytes("200m"), 200))
+class QuotaManager(celebornConf: CelebornConf, configService: ConfigService) extends Logging {
+  val DEFAULT_QUOTA = Quota(
```

Comment:
IMO  move checkQuotaSpaceAvailable from Quota to QuotaManager is more better for quota manage and check, we also can add a configuration at service side to switch quota check enable/disable. It can refactor in next prs.

Reply:
Yea, will done in next pr